gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/management-openapi-v2.yaml

```diff
@@ -145,6 +145,31 @@ paths:
                     $ref: "#/components/responses/ApisResponse"
                 default:
                     $ref: "#/components/responses/Error"
+    /environments/{envId}/apis/_verify-path:
+        parameters:
+            - $ref: "#/components/parameters/envIdParam"
+        post:
+            parameters:
+                - $ref: "#/components/parameters/envIdParam"
+            tags:
+                - APIs
+            summary: Sanitize API paths
+            description: |-
+                Sanitize paths before creating a new API.<br>
+                This will sanitize path and hosts (depending on environment domain restrictions), and will check that path is not already used by other APIs in the environment.<br>
+                It will raise error if path is invalid (contains invalid chars), or if path is already covered by another API.
```

Comment:
Another thought 😁 

The http code is used to validate or invalidate paths, but this seems to me to be incorrect. in this case the request is legitimate. 

perhaps it would be more relevant to return a 200 with an obj like : 

```
{
valid: true | false;
erroMessage?: "..." 
}
```

Reply:
in the `_verify` resource for subscriptions, we use this structure:

```
ok:
    type: boolean
    description: Indicates whether the creation can be done or not.
reason:
    type: string
    description: An optional reason giving details about the result.
```